### ISSUE TITLE
ggml_graph_compute: Assertion `false' failed

### ISSUE THREAD

**Issue by muoten** [ggerganov/whisper.cpp#26]
Examples work fine for me, but I get an error when trying different wavefiles larger than 21s:

```
./main -m models/ggml-base.en.bin mycut-22s-16khz.wav
whisper_model_load: loading model from 'models/ggml-base.en.bin'
[...]
main: processing 'mycut-22s-16khz.wav' (352000 samples, 22.0 sec), 2 threads, lang = en, task = transcribe, timestamps = 1 ...
[...]

main: ggml.c:6658: ggml_graph_compute: Assertion `false' failed.
Aborted (core dumped)
```
It's apparently fixed if I comment the assert and substitute by `cgraph->work = NULL;` in ggml.c. But I guess it's not the best workaround, as it crashes again with sigfault if audio duration is more than 43s aproximately.

Running on Ubuntu 18.04 LTS (GNU/Linux 4.15.0-187-generic x86_64), after fixing CACHE_LINE_SIZE and initializers issues https://github.com/ggerganov/whisper.cpp/issues/11

Any hint? Thanks!

**Comment by ggerganov** [comment]
Hi, I am pretty sure that the suggested initializer fix in #11 is wrong.
You have to use `struct ggml_cgraph gf = {}`:

```c++
struct ggml_cgraph gf = {}; gf.n_threads = n_threads;
```

I've already pushed this fix on `master`, so I think you can simply update to HEAD.

**Comment by muoten** [comment]
It works! Thank you very much!

Amazing work, btw. Kudos :)
